<?xml version="1.0"?>
<launch>
    <arg name="user" default="qtrobot" />
    <arg name="participant" default="1" />
    <arg name="checkpoint" default="dlc8_extended" />

    <node pkg="dlbm_robot" type="wrapper.py" name="dlbm_robot" output="screen">
        <param name="output_directory" type="str" value="/home/$(arg user)/experiments/$(arg participant)/game_performance/" />
        <param name="checkpoint_name" type="str" value="$(arg checkpoint)" />
    </node>
</launch>
